include/SugarEmailAddress/SugarEmailAddress.php

```diff
@@ -1821,6 +1984,7 @@ public static function getSendConfirmOptInEmailActionLinkDefs($module, $returnMo
  */
 function getEmailAddressWidget($focus, $field, $value, $view, $tabindex = '0')
 {
+    // TODO: Move this into an other file eg utils.php
```

Comment:
Do not leave TODO in the code.

Reply:
I will move the function into it's own file.